File: Serene/Serene.Script/Meeting/MeetingAgendaRelevant/MeetingAgendaRelevantDialog.cs

namespace Serene.Meeting
{
    using jQueryApi;
    using Serenity;
    using System.Collections.Generic;

    [IdProperty("MeetingAgendaRelevantId"), IsActiveProperty("IsActive")]
    [FormKey("Meeting.MeetingAgendaRelevant"), LocalTextPrefix("Meeting.MeetingAgendaRelevant"), Service("Meeting/MeetingAgendaRelevant")]
    public class MeetingAgendaRelevantDialog : EntityDialog<MeetingAgendaRelevantRow>
    {
    }
}

File: Serene/Serene.Core/Modules/Meeting/MeetingAgendaRelevant/MeetingAgendaRelevantDialog.ts

namespace Serene.Meeting {

    @Serenity.Decorators.registerClass()
    @Serenity.Decorators.responsive()
    export class MeetingAgendaRelevantDialog extends Serenity.EntityDialog<MeetingAgendaRelevantRow, any> {
        protected getFormKey() { return MeetingAgendaRelevantForm.formKey; }
        protected getIdProperty() { return MeetingAgendaRelevantRow.idProperty; }
        protected getLocalTextPrefix() { return MeetingAgendaRelevantRow.localTextPrefix; }
        protected getService() { return MeetingAgendaRelevantService.baseUrl; }

        protected form = new MeetingAgendaRelevantForm(this.idPrefix);

    }
}